src/combinatorics/quiver/basic.lean

```diff
@@ -86,6 +86,12 @@ def comp {U : Type*} [quiver U] {V : Type*} [quiver V] {W : Type*} [quiver W]
 { obj := λ X, G.obj (F.obj X),
   map := λ X Y f, G.map (F.map f), }
 
+@[simp] lemma comp_id {U : Type*} [quiver U] {V : Type*} [quiver V] (F : prefunctor U V) :
+  F.comp (id _) = F := by { cases F, refl, }
```

Comment:
```suggestion
  F.comp (id _) = F := rfl
```
or if it doesn't work
```suggestion
  F.comp (id _) = F := fun_like.coe_injective rfl
```

Reply:
Neither works. I don't think `prefunctor` has a `fun_like` instance.

Reply:
Ah yeah of course it doesn't 🤦